Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
+RUN apt update && apt install poppler-utils -y
+COPY setup.py setup.cfg README.rst ./
+COPY refextract refextract/
+RUN python setup.py install 
+ENTRYPOINT exec gunicorn -b :5000 --access-logfile - --error-logfile - refextract.app:app --timeout 500
```

Comment:
timeout is in seconds?

Reply:
yes, I made it that big with reference to the stats that we had but I can decrease it

Reply:
According to our results it should be even more I believe, 500s are ~8minutes and we have seen it goes up to 10mins

Reply:
maybe we should run CI tests in the container, otherwise the image is never tested before QA.